scripts/publish_vessel_info

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+source pipe-tools-utils
+
+THIS_SCRIPT_DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
+ASSETS=${THIS_SCRIPT_DIR}/../assets
+ARGS=( \
+  SOURCE_QUERY \
+  TEMP_EXTRACT_TABLE_PREFIX
+  TEMP_BUCKET \
+  ELASTIC_SEARCH_SERVER_URL \
+  ELASTIC_SEARCH_SERVER_AUTH \
+  ELASTIC_SEARCH_INDEX_ALIAS \
+  ELASTIC_SEARCH_INDEX_MAPPINGS \
+)
+
+################################################################################
+# Validate and extract arguments
+################################################################################
+display_usage() {
+  echo -e "\nUsage:\n$ SOURCE_QUERY TEMP_EXTRACT_TABLE_PREFIX TEMP_BUCKET ELASTIC_SEARCH_SERVER_URL ELASTIC_SEARCH_SERVER_AUTH ELASTIC_SEARCH_INDEX_ALIAS ELASTIC_SEARCH_INDEX_MAPPINGS\n"
+}
+
+if [[ $# -ne ${#ARGS[@]} ]]
+then
+    display_usage
+    exit 1
+fi
+
+ARG_VALUES=("$@")
+PARAMS=()
+for index in ${!ARGS[*]}; do
+  echo "${ARGS[$index]}=${ARG_VALUES[$index]}"
+  declare "${ARGS[$index]}"="${ARG_VALUES[$index]}"
+done
+
+
+CURRENT_TIMESTAMP=$(date -u +%Y%m%d%H%M%S)
+TEMP_PATH=gs://${TEMP_BUCKET}/pipe-vessels/${CURRENT_TIMESTAMP}
+
+echo "Running $0"
+echo "  SOURCE_QUERY: $SOURCE_QUERY"
+echo "  TEMP_EXTRACT_TABLE_PREFIX: $TEMP_EXTRACT_TABLE_PREFIX"
+echo "  TEMP_BUCKET: $TEMP_BUCKET"
+echo "  ELASTIC_SEARCH_SERVER_URL: $ELASTIC_SEARCH_SERVER_URL"
+echo "  ELASTIC_SEARCH_SERVER_AUTH: [HIDDEN]"
+echo "  ELASTIC_SEARCH_INDEX_ALIAS: $ELASTIC_SEARCH_INDEX_ALIAS"
+echo "  ELASTIC_SEARCH_INDEX_MAPPINGS: $ELASTIC_SEARCH_INDEX_MAPPINGS"
+################################################################################
+# Running bigquery query to generate vessel information
+################################################################################
+echo "Running vessel information query"
+EXTRACT_TABLE="${TEMP_EXTRACT_TABLE_PREFIX}_${CURRENT_TIMESTAMP}"
+bq query \
+  --destination_table ${EXTRACT_TABLE} \
```

Comment:
I would add `--headless` and `--allow_large_results` to `bq query`. This may work for indo, but may not scale for others.